dl/src/main/scala/com/intel/analytics/bigdl/dataset/Types.scala

```diff
@@ -102,6 +104,7 @@ class Sample[T: ClassTag] (
 
   def copyToLabel(storage: Array[T], offset: Int, length: Int): Unit = {
```

Comment:
I think it's more confusing. extractLabel or extractFeature should be what it means here?